### DISCUSSION TITLE
Thanks for the great work.

### DISCUSSION THREAD

**Discussion by marnux** [roelderickx/connective-plugin-linux#30] [General]
I found this by accident and it is a life saver for Doccle. It is so annoying that they don't make this work easily on linux. The effort seems relatively small, based on the implementation i have seen here. Even the BE government provides a linux supported solution for eid authentication.
I have started reaching out to people within my company to dump Doccle and go somewhere else.

**Comment by roelderickx** [comment]
I'm pretty sure the government's eid-mw can be used on proprietary sites as well. I investigated their codebase and I can assure you it is pretty solid. Which makes me wonder why we need Connective, or Itsme, or ... It seems the same technology is reinvented all the time, for me that's what really is annoying.
So be careful what you wish for, the alternative to Doccle may be even worse ;-)
